sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -312,12 +362,14 @@ public ReadFiles withSplit() {
     static class SplitReadFn extends DoFn<FileIO.ReadableFile, GenericRecord> {
       private Class<? extends GenericData> modelClass;
       private static final Logger LOG = LoggerFactory.getLogger(SplitReadFn.class);
+      private String requestSchemaString;
```

Comment:
Is there any reason to use `String` instead of `Schema`? Looks like this is referred only once.

Reply:
Because the Schema class is not serializable so can only pass with string.